discovery.md

```diff
@@ -538,6 +539,10 @@ This MUST return an array of zero or more Services. The array MUST contain all
 Services available via this Discovery Enpoint. Any Service previously returned
 to a client that does not appear in this result can be assumed to be deleted.
 
+The collection of services MAY be filtered by supplying an attribute as a query
+parameter.  Doing so will cause only Services with an exact match for that
+attribute to be included in the result.
```

Comment:
This replaces the `GET /services?name={name}` path with a more general capability that is consistent with RESTful practices.

Reply:
I think the intent was that we didn't want to force people to support queries across all possible attributes. I may have missed it, but did you define the list of attributes that MUST be supported?

Reply:
I did not specify a list.  Given the beginning state it seems `name` is the minimum supported set of filters.  I'll clarify, thank you for the gentle reminder of that intention.

Reply:
Please let me know if [69359c3](https://github.com/cloudevents/spec/pull/837/commits/69359c303c7733345fdde4da1fb78b42a1478058) doesn't fully resolve this.